src/qz/ws/PrintSocketClient.java

```diff
@@ -159,6 +161,8 @@ public void onClose(Session session, int closeCode, String reason) {
 
     @OnWebSocketError
     public void onError(Session session, Throwable error) {
+        if (error instanceof EOFException) return;
```

Comment:
@Vzor- claims the `java.io.EOFException` swallows a scenario where the connection is denied.

Reply:
We shouldn't need to catch both since the jetty eof extends the javaio one
But I also think we should only be checking for the lower level jetty one anyway, since its specifically for this strange websocket closing case, whereas the javaio one might involve other cases we'd want to log.

Reply:
EOFException Should be enough, originally I was checking the class more explicitly but instanceof should catch both with EOFException.

Reply:
To reproduce the issue, attempt to connect anonymously (I just did sample.html), when the allow/deny connection-from-localhost dialog comes up, hit deny. On win10 firefox and edge I get  `java.io.EOFException: Disconnected`